panel/src/components/Views/ModelView.vue

```diff
@@ -53,7 +55,7 @@ export default {
 			return this.lock.user.email;
 		},
 		hasChanges() {
```

Comment:
`Suggestion:` maybe should be called `hasDiff` for consistency

Reply:
I've refactored it, which involved quite some steps. Could you take a closer look at it and check if I didn't miss anything? Do you think it's better this way?